packages/ui-kit/src/components/Counter/Counter.stories.tsx

```diff
@@ -73,12 +82,26 @@ const styles = {
     margin: 0;
     font-weight: 500;
     color: #2e90fa;
+  `,
+  cardComparisonTitle: css`
+    margin: 0;
+    font-weight: 500;
+    color: #7d89b0;
+  `,
+  cardComparisonValues: css`
+    display: flex;
+    align-items: flex-end;
+    color: #7d89b0;
+  `,
+  cardComparisonFrom: css`
+    margin: 0 0 4px 1ch;
   `
 }
 
 export const SingleValueStory: Story = {
   name: 'Single value',
-  args: connectedParams
+  args: connectedParams,
+  render: (args) => <Counter {...args} />
```

Comment:
With the current implementation, it is required to explicitly pass the `render` prop in order to make[ the custom decorator work](https://github.com/propeldata/ui-kit/pull/127/files#diff-1f24638e3f28bc5c8eae32ea7e5d2fc132df223ccdd7fcb7100dc8e97b027155R32).